### PR TITLE
[incubator/jaeger] Added option to enable elasticsearch-curator for cleaning old elasticserch data

### LINKED ISSUE #8956: Jaeger chart doesn't include cronjob to run jaegertracing/jaeger-es-index-cleaner
**Is this a request for help?**:

No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Feature request

I'd like to run the es-index-cleaner script that appears to be available in this image: https://hub.docker.com/r/jaegertracing/jaeger-es-index-cleaner/

I haven't yet tested this specific image extensively, but it appears to be a way to run esCleaner.py as described here: https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/es

I'm planning on adding a cronjob template to the Jaeger Helm chart, but I figured before I do so I'd check with the community to see that this would be desired? Otherwise, I'll just run this independent of the Jaeger helm chart.

If it is, it appears that there is kind of a model to follow in the Spark cronjob included in the current image. I'll model this after that.

Again, I'm happy to do this this work, just gauging interest in my contributing this feature...

### PR DESCRIPTION
**Pull request by Towmeykaw**
Signed-off-by: towmeykaw <tkindmark@gmail.com>

#### Which issue this PR fixes
  - fixes #8956

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: incubator/jaeger/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.4.1
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.7.0
+version: 0.7.1
```
Comment (cpanato) [inline incubator/jaeger/Chart.yaml]: adding new feature, please bump to `0.8.0`

Comment (Towmeykaw) [reply incubator/jaeger/Chart.yaml]: Done

**Comment by idealhack** [comment]
can we reopen this?

/reopen

**Pull request closed without merging**